modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -575,6 +575,50 @@ public Comment getByExternalReferenceCodeComment(
 			serviceBuilderComment, _commentManager, PortalUtil.getPortal());
 	}
 
+	@Override
+	public Page<Comment> getByExternalReferenceCodeCommentChildCommentsPage(
+			String externalReferenceCode, String commentExternalReferenceCode,
+			String search, Aggregation aggregation, Filter filter,
+			Pagination pagination, Sort[] sorts)
+		throws Exception {
+
+		if (!_objectDefinition.isEnableComments() ||
+			!FeatureFlagManagerUtil.isEnabled(
+				_objectDefinition.getCompanyId(), "LPD-69419")) {
+
+			throw new UnsupportedOperationException();
+		}
+
+		ObjectEntry objectEntry = _getObjectEntry(externalReferenceCode, null);
+
+		com.liferay.portal.kernel.comment.Comment serviceBuilderComment =
+			_fetchComment(
+				ObjectEntry.class.getName(), objectEntry.getId(),
+				commentExternalReferenceCode,
+				_getNonzeroGroupId(objectEntry.getId()));
+
+		if (serviceBuilderComment == null) {
+			throw new NotFoundException();
+		}
+
+		return CommentResourceUtil.getComments(
+			HashMapBuilder.put(
+				"create",
+				addAction(
+					ActionKeys.ADD_DISCUSSION,
+					"postByExternalReferenceCodeCommentChildComment",
+					ObjectEntry.class.getName(), null)
+			).put(
+				"delete",
+				addAction(
+					ActionKeys.DELETE, "deleteByExternalReferenceCodeComment",
+					ObjectEntry.class.getName(), null)
+			).build(),
+			serviceBuilderComment.getCommentId(), contextCompany.getCompanyId(),
```

Comment:
Are these links being created correctly? I see an ERC endpoint, but we're passing the CommentId below.